osmnx/simplify.py

```diff
@@ -325,11 +334,20 @@ def clean_intersections(G, tolerance=15, dead_ends=False):
 
     Returns
     ----------
-    intersection_centroids : geopandas.GeoSeries
-        a GeoSeries of shapely Points representing the centroids of street
-        intersections
+    G__ : networkx multidigraph
+        a new spatial graph, topologically equal to G (approximately), but with
+        clusters of nodes merged into a single centroid node.
     """
 
+    start_time = time.time()
+
+    if 'proj' not in G.graph['crs'] and G.graph['proj'] != "utm":
```

Comment:
Couldn't there also just be the 'init' key in G.graph['crs']?

Reply:
At the start of the method, I wanted to test if the graph is projected, otherwise the method won't work as excepted. In my view, an exception should be thrown if that is the case, but I just logged as a warning and moved on.

But I guess you're right, the init key can have some value that specifies that the graph is projected, rather than the default `epsg:4326`. But can that happen in the standard osmnx program flow? You always get an unprojected graph from methods like `graph_from_bbox,` and you need to project it through `ox.project_graph`, thus setting `G.graph['crs']['proj'] = 'utm'`. Or do I have this wrong?

I've also realised that that statement is also wrong as it stands. It should be:

```python
if 'proj' not in G.graph['crs'] or G.graph['crs']['proj'] != "utm":
```

Is there a better way of checking if a graph is unprojected?

Reply:
It might be best to check if it's projected and, if not, issue a warning to the user but proceed anyway. The docstring should make clear that cleaning will occur in the graph's current units (degrees, meters, feet, whatever they may be---but user is advised to not use unprojected/degrees).

But, we shouldn't force them to use a UTM projection. Although the default is UTM, OSMnx allows users to project to any CRS of their choice. So we don't want to hard code anything that requires UTM or that requires meters (as other CRSs can use other units).

To check if the graph differs from the default projection, you can use:

```python
G.graph['crs'] == ox.settings.default_crs
```

Reply:
Yes, that is sensible. Thanks, I'll update the statement and docstring.